spec/support/shared_context/siade_context.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.shared_context 'with siade payloads' do
```

Comment:
c'est du détail mais j'aurais renommé

```suggestion
RSpec.shared_context 'siade payloads' do
```

c'est plus claire `include_context 'siade payload'` qu'avec le with

Reply:
Non on garde la reco rubocop imo.